composer.json

```diff
@@ -20,19 +20,18 @@
         "forum": "https://discourse.zendframework.com/c/questions/expressive"
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.2.0",
+        "php": "^7.1",
         "psr/container": "^1.0",
         "psr/http-message": "^1.0.1",
-        "zendframework/zend-expressive-helpers": "^1.4 || ^2.2 || ^3.0.1 || ^4.0",
-        "zendframework/zend-expressive-router": "^1.3.2 || ^2.1",
-        "zendframework/zend-expressive-template": "^1.0.4",
+        "zendframework/zend-expressive-helpers": "^5.0.0-dev",
+        "zendframework/zend-expressive-router": "^3.0.0-dev",
+        "zendframework/zend-expressive-template": "^2.0.0-dev",
         "zendframework/zend-servicemanager": "^2.7.8 || ^3.3",
```

Comment:
Should we update to last hotfixes, I mean 2.7.10 and 3.3.1 ?

Reply:
I don't see any compelling reason to; the main thing here is API compatibility.